addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/discovery/MaxCubeBridgeDiscovery.java

```diff
@@ -44,9 +46,19 @@
 	private final static Logger logger = LoggerFactory.getLogger(MaxCubeBridgeDiscovery.class);
 
 	static boolean discoveryRunning = false;
-
+	
+	/** The refresh interval for discovery of MAX! Cubes */
+	private long refreshInterval = 20;
```

Comment:
Is 20 secs not too short? I have no idea how much network load a scan means, but doing too many broadcasts might flood the network. But I leave the value of this setting to your expertise :-)